src/components/Temporary/ThemeSection.tsx

```diff
@@ -0,0 +1,236 @@
+  const periwinkleChange = useTransform(scrollYSpring, [0, 1], [digitalRedHex, colorNameToHex.periwinkle]);
+  const robinsEggChange = useTransform(scrollYSpring, [0, 1], [digitalRedHex, colorNameToHex['robins-egg']]);
+
+  return (
+    <div>
+      <Container
+        bgColor="black"
+        pt={9}
+        className="su-relative su-overflow-hidden"
+      >
+        <Text size={2} leading="tight" font="serif">Themes</Text>
+        <Heading as="h2" size="splash" leading="none" font="druk" className="su-max-w-1000">
+          Square pegs, huge goals.
+        </Heading>
+        <FlexBox className="su-mx-auto" justifyContent="center">
+          <m.svg ref={svgRef} className="su-shrink-0" width="140" height="375" viewBox="0 0 140 375" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <m.path
+              d="M1.00142 0.0948767L2.56929 193.733C3.3767 293.452 64.5865 373.801 139.284 373.196"
+              stroke="white"
+              strokeWidth="2"
+              vectorEffect="non-scaling-stroke"
+              initial={{ pathLength: 0 }}
+              animate={{ pathLength: introCurveInView ? 1 : 0 }}
+              transition={{
+                delay: 0.3,
+                duration: 0.6,
+                ease: 'easeInOut',
+              }}
+            />
+          </m.svg>
+          <AnimateInView animation="slideDown" delay={0.6}>
+            <Paragraph variant="overview" font="serif" className="su-max-w-[100rem] su-rs-mt-8 su-rs-mb-4 su-ml-[3rem] su-pr-[17rem]">
+              These aren’t priorities to be funded—they are a promise we make to each other.
+              To build a more ethical future, we need to move concern for others from being an afterthought
+              to being integral to the thought process. In every field of study,
+              we need to ensure consequences are weighed and tradeoffs are made on purpose. This is how we get there.
+            </Paragraph>
+          </AnimateInView>
+        </FlexBox>
+        <div className={dcnb('su-sr-only focus-within:su-not-sr-only su-w-fit children:su-mx-auto', !shouldAnimate ? 'su-hidden' : '')}>
+          <CtaButton onClick={() => setShouldAnimate(false)}>Reveal themes visually</CtaButton>
+        </div>
+        {/* This grid contains the 4 animated lines behind the theme cards */}
+        <Grid md={2} className="su-absolute su-left-0 su-top-[180rem] su-w-full su-gap-y-[30rem]">
+          <div className="su-max-w-full su-overflow-hidden">
+            <m.svg className="su-mr-0 su-ml-auto" viewBox="0 0 952 461" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <m.path
+                d="M951.043 1H450.62C192.915 1 -16 206.517 -16 460.032"
+                stroke="white"
+                strokeWidth="2"
+                vectorEffect="non-scaling-stroke"
+                style={{ pathLength: curveDownSpring }}
+              />
+            </m.svg>
```

Comment:
This is how we turn SVG into animated paths. I added the `vectorEffect="non-scaling-stroke"` so that when the SVG gets resized due to parent container size, the stroke width (2px) is preserved.